synapse/lib/aha.py

```diff
@@ -250,26 +253,31 @@ async def delAhaSvc(self, name, network=None):
         '''
         Remove an AHA service entry.
         '''
+        if network is not None:
+            s_common.deprecated('delAhaSvc() network argument', curv='v2.206.0')
         svcname, svcnetw, svcfull = self.cell._nameAndNetwork(name, network)
         await self._reqUserAllowed(('aha', 'service', 'del', svcnetw, svcname))
         return await self.cell.delAhaSvc(name, network=network)
 
     async def getCaCert(self, network):
-
+        s_common.deprecated('genCaCert() network argument', curv='v2.206.0')
```

Comment:
```suggestion
        s_common.deprecated('getCaCert()', curv='v2.206.0')
```

Reply:
Or should this be checking if `network` is not None?

Reply:
In 3.x.x `getCaCert()` on its own should return the CA certificate for AHA cell's `aha:network` value. So it's kind of an odd one to handle :thinking:

Reply:
tweaked the language a bit